scanners/ios/workflow.go

```diff
@@ -15,24 +15,29 @@ const (
 	AutomaticCodeSigningValue             = "api-key"
 	CacheLevelKey                         = "cache_level"
 	CacheLevelNone                        = "none"
-)
 
-const primaryTestDescription = `The workflow executes the tests. The *retry_on_failure* test repetition mode is enabled.`
+	// test workflow
+	primaryWorkflowID = "primary"
```

Comment:
I thought almost though that the existence of the `primaryWorkflowID` and `deployWorkflowID` constants is an error but then realised that they are still used with macOS. I would propose to completely remove these and update the Mac scanner as well. I have two options for it:
1.)
Just reuse the same `run_tests`, `build` and `archive_and_export_app` because those workflows do almost the same thing. Expect the `build` one because we do not have a build step on Mac so that one does nothing useful. But still the intention would be the same so I can justify having the same name.

2.)
The second option is to also reuse the same description too. The `run_tests` and `build` can be reused as they are. The `archive_and_export_app` needs a separate one because instead of an ipa we have an exported app package.

Reply:
For now, the aim is updating the ios and android workflows only.